client/a8c-for-agencies/sections/overview/body/products/style.scss

```
@@ -0,0 +1,4 @@
+.a4a-overview-products__woocommerce-logo {
+	fill: var(--studio-woocommerce-purple-60);
```

```suggestion
	fill: var(--color-woocommerce);
```
Could use the `--color` naming convention here, although under the hood `--color-woocommerce` just maps to `--studio-woocommerce-purple` so... 😆

---

Didn't know that! Thanks!